.github/workflows/Tests.yaml

```diff
@@ -245,10 +245,16 @@ jobs:
             compiler: gcc-9
           - use_pch: OFF
             build_type: Debug
-            compiler: clang-5.0
+            compiler: gcc-10
```

Comment:
you didn't add gcc-10 to the build matrix yet. I don't mind adding it here already, just wanted to make sure it was intentional.

Reply:
Yes, it was. Once we've fixed building with GCC10 it will be easy to add